spec/unit/provider/registry_key_spec.rb

```diff
@@ -289,6 +297,8 @@
     let(:testval1_wrong_type) { { :name => "one", :type => :string, :data => 255.chr * 1 } }
     let(:testval1_wrong_data) { { :name => "one", :type => :binary, :data => 254.chr * 1 } }
     let(:testval2) { { :name => "two", :type => :binary, :data => 0.chr * 1 } }
+    let(:testval3) { { :name => "three", :type => :binary, :data => 0.chr * 1 } }
+    let(:testval3_data_string) { { :name => "three", :type => :binary, :data => 0.chr * 1 } }
```

Comment:
if we're not doing anything with binary cases, we should move the case into an appropriate context rather than having the test run for other cases that are meaningless.